### PR TITLE
move database migration to first boot

### PR DESCRIPTION
**Pull request by gabriel-v**
closes https://github.com/liquidinvestigations/liquidinvestigations/issues/87

### CODE REVIEW

File: ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh [hunk 1]
```diff
@@ -1,3 +1,8 @@
 #!/bin/bash
 
-echo "liquid core was born ready"
+set -e
```
Comment (mgax) [inline ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh]: You can fold this line into the shebang line: `#!/bin/bash -e`

File: ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh [hunk 2]
```diff
@@ -1,3 +1,8 @@
...
+set -x
+
+cd /opt/liquid-core/liquid-core
+../venv/bin/python ./manage.py migrate
+../venv/bin/python ./manage.py collectstatic --noinput
```
Comment (mgax) [inline ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh]: Eh, this should actually be done during setup, those static files aren't going to change.

File: ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh (hunk 1 above)
Comment (gabriel-v) [reply ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh]: If I fold the `-e`, it'll be the only file with a folded `-e` in the shebang. If we change all of them like that, we'll need to get used to spotting it there, too.

And that will be one more place we need to be watching closely when we're debugging these bash scripts.

File: ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh (hunk 2 above)
Comment (gabriel-v) [reply ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh]: Right, committing fix now

File: ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh (hunk 1 above)
Comment (mgax) [reply ansible/roles/liquid-core/templates/initialize.d/liquid-core.sh]: Yes, I stand corrected, the `set -e` is more readable. Forget that I even mentioned it.

**Pull request merged**